tasks/libs/github_jira_map.yaml

```diff
@@ -16,7 +16,7 @@
 '@datadog/metrics-aggregation': AGGR
 '@datadog/serverless': SVLS
 '@datadog/remote-config': RC
-'@datadog/fleet': RC
+'@datadog/fleet': FA
```

Comment:
Should you also add a section in the [ddqa](https://github.com/DataDog/datadog-agent/blob/main/.ddqa/config.toml#L113-L118) file as well?